chapter2/tests/t5/chapter2-l2-t5-get-file-unique-extensions.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
```

Comment:
No messages to user when folder doesn't exist, not user friendly again :)

![image](https://user-images.githubusercontent.com/26448672/230022512-c96e2a46-0a39-4f22-8c16-68ef4799fca0.png)